images/image-inspector.yml

```diff
@@ -1,7 +1,3 @@
-# <lmeyer> I don't believe anything needs this to be built and released
-# regularly. this is used by cloudforms and the version should correspond to
-# the RPM it installs.  so, let's not re-version and build it.
```

Comment:
is there an automated way to keep the comments? Seems like they might sometimes be helpful as to "why is this the way it is"

Reply:
Setting required rewrites the contents. It's because it does load > edit > save. Which always rewrites the content.
Regarding comments: No. For same reason as above. pyyaml doesn't support comments nor does anything I've found. We would have to switch to something like HSON which supports comments and comment parsing.

Reply:
@sosiouxme I stand corrected -> https://pypi.org/project/ruamel.yaml/
That supposedly supports it, but we'd have to change doozer over to using that for everything.

Reply:
Though on further thought... this would require much more rework that I'd rather @sosiouxme 
The problem is that when we load in the data it gets mangled into a variety of other objects before being saved back out. Looking at `ruamel`, we'd have to replace all those internal objects with pure `ruamel` ones.
Yes, we could do these changes by hand... but I really would rather not. As it is, ocp-build-data could at any time move over to being 100% generated by other sources if the OKD guys wanted to move in that direction. It really needs to stay pure data with no expectation that comments will ever be valid or stick around. @tbielawa thoughts?

Reply:
No need for that much work; we can treat this as just data. "Why is it this way?" would probably be better to track somewhere else, though I'm not exactly sure where (git commit msgs don't seem quite adequate).